### PR TITLE
Feature/3ds2block

### PR DESCRIPTION
**Pull request by Aleffio**
**Description**
adding missing block from previous commit,
adding screenHeight to OneClick, necessary for ChallengeShopper to go through.
Fixing bug for OneClick about browserinfo fields
Try catch on payments/details call,
cancel the order in case of exception,
console.log on the onError callback, risk enabled false

### CODE REVIEW

File: app/design/frontend/base/default/template/adyen/threeds2.phtml [hunk 1]
```diff
@@ -37,8 +37,7 @@
                             processThreeDS2(result.data);
                         },
                         onError: function () {
-                            //todo verify error
-                            processThreeDS2(null, "error on IdentifyShopper");
+                            console.log("error on IdentifyShopper");
```
Comment (rikterbeek) [inline app/design/frontend/base/default/template/adyen/threeds2.phtml]: please log the result of the onError. see m2

File: app/design/frontend/base/default/template/adyen/threeds2.phtml [hunk 2]
```diff
@@ -49,8 +48,7 @@
                             processThreeDS2(result.data);
                         },
                         onError: function () {
-                            //todo verify error
-                            processThreeDS2(null, "error on ChallengeShopper");
+                            console.log("error on ChallengeShopper");
```
Comment (rikterbeek) [inline app/design/frontend/base/default/template/adyen/threeds2.phtml]: please log the result of the onError. see m2

File: app/design/frontend/base/default/template/adyen/threeds2.phtml (hunk 1 above)
Comment (Aleffio) [reply app/design/frontend/base/default/template/adyen/threeds2.phtml]: thanks, done

File: app/design/frontend/base/default/template/adyen/threeds2.phtml (hunk 2 above)
Comment (Aleffio) [reply app/design/frontend/base/default/template/adyen/threeds2.phtml]: thanks, done

**Pull request merged**